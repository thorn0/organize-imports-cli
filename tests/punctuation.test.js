const runCli = require("./run-cli");

describe("with --ignore-punctuation", () => {
  runCli("cli/punctuation", ["--list-different", "--ignore-punctuation", "file1.js", "file2.js"]).test();
});

describe("without --ignore-punctuation", () => {
    runCli("cli/punctuation", ["--list-different", "file1.js", "file2.js"]).test();
  });
  