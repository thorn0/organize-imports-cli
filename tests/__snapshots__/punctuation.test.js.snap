// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`with --ignore-punctuation (status) 1`] = `0`;

exports[`with --ignore-punctuation (stderr) 1`] = `""`;

exports[`with --ignore-punctuation (stdout) 1`] = `""`;

exports[`with --ignore-punctuation (write) 1`] = `Array []`;

exports[`without --ignore-punctuation (status) 1`] = `2`;

exports[`without --ignore-punctuation (stderr) 1`] = `""`;

exports[`without --ignore-punctuation (stdout) 1`] = `
"<dir>/file1.js
<dir>/file2.js
"
`;

exports[`without --ignore-punctuation (write) 1`] = `Array []`;
